src/templates/home-page/Hero/index.tsx

```diff
@@ -49,13 +49,23 @@ const Hero: FC<{
               spacing={12}
             >
               {ctaButtons.map(
-                (_: { label: string; url: string }, index: number) => {
+                (
+                  _: { label: string; url: string; posthogLabel: string },
+                  index: number
+                ) => {
+                  const additionalProps = {} as any
+                  if (_.posthogLabel) {
+                    additionalProps["data-ph-capture-attribute-button-name"] =
+                      _.posthogLabel
+                  }
+
                   return (
                     <ExternalButtonLink
                       key={_.label}
                       {...heroButtonProps}
                       variant={index === 0 ? "special" : "outline"}
                       href={_.url as ExternalLinkHref}
+                      {...additionalProps}
```

Comment:
Looks like we should extract this part to function or component to handle common logic.

Reply:
b9153dee2218caa8dddc0df9182cb61d2583cef3